sample_app/sampleapp_common.c

```diff
@@ -1083,6 +1138,28 @@ static int app_new_data_status_ind (
          arep,
          changes,
          data_status);
+      printf (
+         "   %s, %s, %s, %s, %s\n",
+         is_running ? "Run" : "Stop",
+         is_valid ? "Valid" : "Invalid",
+         (data_status & BIT (PNET_DATA_STATUS_BIT_STATE)) ? "Primary"
+                                                          : "Backup",
+         (data_status & BIT (PNET_DATA_STATUS_BIT_STATION_PROBLEM_INDICATOR))
+            ? "Normal operation"
+            : "Problem",
+         (data_status & BIT (PNET_DATA_STATUS_BIT_IGNORE))
+            ? "Ignore data status"
+            : "Evaluate data status");
+   }
+
+   if (is_running == false || is_valid == false)
+   {
+      if (p_appdata->arguments.verbosity > 0)
+      {
+         printf ("Setting outputs to default value (due to new data status "
```

Comment:
Remove if printout is moved to app_set_outputs_default_value()

Reply:
Done